pulsar-broker-common/src/main/java/org/apache/pulsar/broker/ServiceConfiguration.java

```diff
@@ -323,6 +323,12 @@ public class ServiceConfiguration implements PulsarConfiguration {
             + "(0 to disable limiting)")
     private int maxHttpServerConnections = 2048;
 
+    @FieldContext(
+            category = CATEGORY_SERVER,
+            doc = "enable compression when the HTTP service responds to the client"
+    )
+    private boolean enableCompress = false;
```

Comment:
`enableCompressMetricsData` should be better

Reply:
please review